backend/contributions/tests/test_steward_permissions.py

```
@@ -210,6 +217,102 @@ def test_propose_only_steward_only_sees_pending_permitted_submissions(self):
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        note.refresh_from_db()
+        self.assertEqual(note.message, 'Proposal note')
+
+    def test_steward_cannot_edit_another_stewards_proposal_note(self):
+        """Only the steward who owns the active proposal can edit its note."""
+        other_steward_user = User.objects.create_user(
+            email='other-steward@test.com',
+            address='0x3333333333333333333333333333333333333333',
+            password='testpass123',
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "password"

(S106)